datafusion/common/src/dfschema.rs

```diff
@@ -1327,6 +1332,7 @@ pub fn qualified_name(qualifier: Option<&TableReference>, name: &str) -> String
 
 #[cfg(test)]
 mod tests {
+    use arrow::datatypes::TimeUnit;
```

Comment:
See above about removing this import. If you still want to keep it, seems like the cargo check wants it below the import on the next line.